src/common/dependency-manager/local-service-node.ts

```diff
@@ -1,25 +1,16 @@
-import { DependencyNode, DependencyNodeOptions } from '../../dependency-manager/src';
+import { DependencyNodeOptions, ServiceNode, ServiceNodeOptions } from '../../dependency-manager/src';
 
 interface LocalServiceNodeOptions {
   service_path: string;
   command?: string;
-  api: {
-    type: string;
-    definitions?: string[];
-  };
 }
 
-export class LocalServiceNode extends DependencyNode implements LocalServiceNodeOptions {
+export class LocalServiceNode extends ServiceNode implements LocalServiceNodeOptions {
```

Comment:
This is going to leave LocalServiceNode with image and tag... but those won't be used, right?

Reply:
The way its implemented at the moment LocalServiceNode needs a tag (latest) for lookups/ref. Image won't be used, but I made it required on the base ServiceNode and this is localized to only the cli so I'm just going to live with it.